staging/src/k8s.io/apimachinery/pkg/util/httpstream/spdy/roundtripper_test.go

```diff
@@ -119,304 +116,300 @@ func localhostCertPool(t *testing.T) *x509.CertPool {
 
 // be sure to unset environment variable https_proxy (if exported) before testing, otherwise the testing will fail unexpectedly.
 func TestRoundTripAndNewConnection(t *testing.T) {
-	for _, redirect := range []bool{false, true} {
```

Comment:
Note to reviewers: large diff from indentation change - recommend hiding whitespace.